commerce_coordinator/apps/rollout/pipeline.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+import waffle
+from openedx_filters import PipelineStep
+
+from commerce_coordinator.apps.commercetools_frontend.constants import COMMERCETOOLS_FRONTEND
+from commerce_coordinator.apps.frontend_app_payment.constants import FRONTEND_APP_PAYMENT_CHECKOUT
+from commerce_coordinator.apps.rollout.constants import REDIRECT_TO_COMMERCETOOLS_CHECKOUT
+
+logger = logging.getLogger(__name__)
+
+
+class GetActiveOrderManagementSystem(PipelineStep):
+    """
+    Determines which order management system to redirect to, based on enterprise role.
+    """
+
+    def run_filter(self, request, active_om_system):
+        """
+        Execute a filter with the signature specified.
+        Arguments:
+            request: request object passed through from the lms filter
+            active_om_system: result from pipeline steps to determine if user has
+                            enterprise role or not & dictact which checkout to redirect to
+        """
+
+        sku = request.query_params.get('sku', '').strip()
+        sku_exists = isinstance(sku, str) and sku  # True if sku not None and is string
```

Comment:
`isinstance(sku, str)` This function would still be True for `''` empty string. We could simply do: 
```suggestion
        sku_exists = True if sku else False
```

Reply:
yes.

Reply:
[Update] We've decided we don't need to explicitly check the instance of the query parameters to be a string as the line above does covers this requirement, so we can simply check the value of `sku` and `course_run` .